test.py

```diff
@@ -1246,6 +1247,11 @@ async def reap(done, pending, signaled):
             if isinstance(result, bool):
                 continue    # skip signaled task result
             console.print_progress(result)
+
+    ms = MinioServer(options.tmpdir, LogPrefixAdapter(logging.getLogger('minio'), {'prefix': 'minio'}))
+    await ms.start()
+    TestSuite.artifacts.add_exit_artifact(None, ms.stop)
+
```

Comment:
By the way, I'd recommend starting MinioServer only if we're running tests that will stress it.

For example, I don't want `./test.py row_cache_stress_test` to start MinioServer.

Reply:
Yeah, that's what came to my mind.

When extending test.py for compaction groups, I introduced this param in boost/suite.yaml:

```
# Enable compaction groups on tests except on a few, listed below
all_can_run_compaction_groups_except:
    - exceptions_optimized_test
    - rate_limiter_test
    - exceptions_fallback_test
```

so we could have a param for tests that need s3 server and the suite would start minio once it first runs a test that require it. but if it's too complicated, we can perhaps postpone this improvement.